fileglancer/handlers.py

```diff
@@ -66,10 +77,17 @@ def get(self):
         self.finish()
 
 
+
 class FileShareHandler(APIHandler):
     """
     API handler for file access using the Filestore class
     """
+    def set_cors_headers(self):
+        self.set_header('Access-Control-Allow-Origin', '*')  # Or your specific domain
+        self.set_header('Access-Control-Allow-Methods', 'GET, POST, PUT, DELETE, PATCH, OPTIONS')
+        self.set_header('Access-Control-Allow-Headers', 'Content-Type, X-XSRFToken')
+        self.set_header('Access-Control-Allow-Credentials', 'true')
```

Comment:
Vibe-coded CORS headers, need to be refined.

Reply:
I am guessing we don't need to the  POST, PUT, DELETE, PATCH verbs. This is a read only end point, correct? Having the wildcard Origin might be necessary if, but if we are only using this to serve up content for our captive neuroglancer, then we can lock it down to that domain.